bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+            implement a specific IoT protocol, data format or IoT platform.
+            The Core Binding Templates specification explains the overall mechanism and requirements for any binding to follow.
+            This document gives implementation guidelines regarding the PROFINET protocol, which is a well-known 
+            high-speed industrial ethernet protocol for communication between industrial control and automation devices. 
+        </p>
+        <p>
+            More specifically, this document defines a set of vocabulary terms that can be used inside a Thing Description document, 
+            and associated rules which allow to describe WoT operations using the PROFINET protocol over the network.
+            Additionally, relevant examples are provided to showcase different vocabulary terms and the associated behavior. 
+        </p>
+    </section>
+    <section id='sotd'>
+        <p>
+            <em>This document is a work in progress</em>
+        </p>
+    </section>
+    <section id='introduction'>
+        <h2>Introduction</h2>
+        <p>
+            PROFINET is an open standard communication protocol developed and published by PROFIBUS Nutzerorganisation (PNO) for Industrial Ethernet. 
+            PROFINET which is an Ethernet version of PROFIBUS is designed to satisfy requirements of most industrial automation processes by providing a 
+            high-speed data transfer for time critical applications like motion control and drive. 
+            It can also have a safety (PROFIsafe) implementation onboard for safety related application. 
+        </p>
+        <p>
+            Generally, profinet is classified into PROFINET IO (Input Output) which is used for distributed input/output processes of field devices (sensors and actuators) 
+            data and PROFINET CBA (Component Based Automation) which is used for distributed automation. The scope of this document focuses on the PROFINET IO. 
+            PROFINET IO network consist of three main actors which are IO controller, IO supervisor and IO device. 
+            The IO controllers are PLCs and DCS, they are most used for profinet cyclic communications like IO data exchange. 
+            The IO supervisor is a standalone application that is mostly used for acyclic communications like diagnostics and parameterization. 
+            The IO device is general field devices that are controlled by both IO controller and IO supervisor.
+        </p>
+        <p>
+            Due to the different requirements by different industrial automation applications, PROFINET IO provides three different communications channels to exchange data between IO controllers and IO device or IO supervisor and IO devices. 
+            The Real-Time (RT) and Isochronous Real Time (IRT) channels are used for cyclic data transfer and high-speed cyclic data transfer for motion control application respectively, 
```

Comment:
RT channel can also support acyclic real time messages, e.g. alarms.

Not sure if IRT also supports acyclic real time.